models/glif_psc.h

```diff
@@ -0,0 +1,354 @@
+/*
+ *  glif_psc.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GLIF_PSC_H
+#define GLIF_PSC_H
+
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+#include "dictdatum.h"
+
+/* BeginDocumentation
+Name: glif_psc - Current-based generalized leaky integrate and fire (GLIF)
+models
+
+Description:
+
+glif_psc is implementations of five generalized leaky integrate and
+fire (GLIF) models [1] with alpha-function shaped synaptic currents.
+Incoming spike events induce a post-synaptic change of current modeled
+by an alpha function. The alpha function is normalized such that an event of
+weight 1.0 results in a peak current of 1 pA at t = tau_syn. On the postsynapic
```

Comment:
While the text here is based on `iaf_psc_[alpha,exp]_multisynapse`, it is not really good in the original. So I would suggest to replace everything from "On the postsynaptic ..." to " ... in the connectors." with the following paragraph

"By default, glif_psc has a single synapse that is accessible through receptor_port 1. An arbitrary number of synapses with different time constants can be configured by setting the desired time constants as tau_syn array. The resulting synapses are addressed through receptor_port 1, 2, 3, ...."

@jessica-mitchell I am not entirely happy with my text either, any suggestions? Once we have a good text, we also need to update `iaf_psc_[alpha,exp]_multisynapse` correspondingly.